meta/runtime.yml

```diff
@@ -46,4 +46,4 @@ plugin_routing:
       redirect: splunk.es.splunk_adaptive_response_notable_event
     adaptive_response_notable_events:
       redirect: splunk.es.splunk_adaptive_response_notable_events
-requires_ansible: ">=2.9.10"
```

Comment:
why are we changing this this needs a major release

Reply:
Will change this back